### DISCUSSION TITLE
正文上下页边距与教务处要求不符

### DISCUSSION THREAD

**Discussion by CrazyElements** [sjtug/SJTUThesis#1036] [提问] [answered]
当前错误的页边距
![db048c0b09218c1b89d1f8b82808723](https://github.com/sjtug/SJTUThesis/assets/105211051/5c6d255b-9631-4561-98c9-5c32460043d2)
正确的页边距 — 顶格写
![cd5301e4a23d5b9a7ca45862d5be25d](https://github.com/sjtug/SJTUThesis/assets/105211051/b1ba0e4f-d284-4eca-86f4-3ef25445cd9d)

请问需要如何更改？

**Comment by NemoYuan2008** [comment]
你好，我无法复现你的问题，从 GitHub 上下载到的模板直接编译出来的结果如下：
![image](https://github.com/sjtug/SJTUThesis/assets/34743600/32edc032-b3f2-4756-98f6-522836c44cff)
你是否引入了可能改变页面布局的宏包？

**Reply by CrazyElements** [reply, marked as answer]
谢谢您，我也试了下从 GitHub 上的 SJTUThesis 2.0.3，它确实是没有问题的。这应该是教务处提供的模板的问题。

无语…… 所以请问要怎么才能修改页边距啊

---
确实是引入了 geometry 宏包

破案了，教务处把 `sjtuthesis.cls` 的页边距设置给修改了，还贴心地给出了正确页边距的注释。
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/3a4c993e-cb9f-46bf-9ac7-2d7c68e574eb)

真的气晕了
